newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationDialog.js

```diff
@@ -218,7 +215,9 @@ const PrivateAssetPackDialog = ({
                           <PriceTag value={prices[0].value} />
                           {getBuyButton(i18n)}
                         </Line>
-                        <Text noMargin>{assetPack.longDescription}</Text>
+                        <Text size="body2" displayInlineAsSpan>
+                          <MarkdownText source={assetPack.longDescription} />
```

Comment:
isn't using markdowntext inside a text component creating warnings in the DOM? (because of p inside h4 or something like that)

I believe this is wider issue that we should fix at some point.
I've seen multiple warnings in the app when markdown text is used.

Reply:
Yes indeed. That's why there's the displayInlineAsSpan, it does not raise errors